setting-up-appsmith/README.md

```diff
@@ -0,0 +1,18 @@
+---
+description: Appsmith stands for speed and getting started with Appsmith is just as fast.
+---
+
+# Setting up Appsmith
+
+You can begin using Appsmith via our cloud instance or by deploying Appsmith yourself
+
+* [Using Appsmith Cloud](./appsmith-cloud.md) **\(recommended\):** Create a new application with just one click
+* [Using Docker](./docker.md): Deploy anywhere using docker
+* [Heroku](./heroku.md)
```

Comment:
I think the following bullet point will look more consistent -
- Appsmith Cloud (instead of Using Appsmith Cloud)
- Docker (instead of Using Docker)
- Heroku